model_compression_toolkit/target_platform_capabilities/target_platform/__init__.py

```diff
@@ -14,9 +14,9 @@
 # ==============================================================================
 
 from model_compression_toolkit.target_platform_capabilities.target_platform.targetplatform2framework.attribute_filter import AttributeFilter
```

Comment:
I want to get rid of the model_compression_toolkit/target_platform_capabilities/target_platform level in the hierarchy, there is no need for it anymore since there is only the /targetplatform2framework/ in it.
Is it too complicated for this huge PR or do you think it would be fine to include this here as well?